Makefile.am

```diff
@@ -36,6 +36,18 @@ EXTRA_DIST = 			\
 	$(man_MANS)		\
 	$(NULL)
 
+# Distribute the Meson build system files as well
+EXTRA_DIST += \
+	help/LINGUAS			\
+	help/meson.build		\
+	meson.build			\
+	meson_options.txt		\
+	meson_post_install.py		\
+	po/meson.build			\
+	src/meson.build			\
+	src/skey/meson.build		\
+	subprojects/mate-submodules.wrap
```

Comment:
Thx for helping but do we need mate-submodules.wrap in tarball?
All files from submodules are included in the tarball.
Before running autogen.sh we have to execute `git submodule init && git submodule update --remote --recursive`, otherwise configure doesn't run.

Reply:
hum… I don't get `submodules/mate-submodules/` in the tarball I create with `make dist`, and somehow `meson.build` doesn't pick up the one in `src/mate-submodules`, so without it I get:
```
Run-time dependency mate-submodules found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency mate-submodules

meson.build:156:24: ERROR: Neither a subproject directory nor a mate-submodules.wrap file was found.
```

Reply:
Maybe there's a better solution for this, but I'm not knowledgeable enough about meson to fix it -- because it does seem to be an issue with the meson build rather than distributing the files.  FWIW, I mostly based the files to distribute off bc13d9f3af1d5114e47b75633c4fd5578214b460, after only including the `meson.build` files and seeing I missed a few :)

Reply:
`make distcheck` is needed to create the tarball.
After  `git submodule init && git submodule update --remote --recursive && ./autogen.sh && make && make distcheck` i have a tarball with the src/mate-submodules/ direction.
That's with current master

Reply:
I tested PR without `subprojects/mate-submodules.wrap` line in make file and it works.

Reply:
@raveit65 hum, I'll test again, but the [release tarball](https://github.com/mate-desktop/mate-media/releases/download/v1.28.0/mate-media-1.28.0.tar.xz) does *not* include the `subprojects` top-level directory.

Reply:
Ok, meson itself needs the file.
```
Run-time dependency mate-submodules found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency mate-submodules

meson.build:156:24: ERROR: Neither a subproject directory nor a mate-submodules.wrap file was found.
```
Than  your PR LGTM :-)

Reply:
@raveit65 do you by any chance have mate-submodules installed so meson finds it?  Or just, what is the output of `meson setup _build .` *from inside a release tarball source tree*?

Reply:
> Ok, meson itself needs the file.

Yeah, but that might be wrong, as it doesn't seem to be looking in the right location if we already have gitmodules putting it some place else…
But again, that's a meson fix for which maybe @yetist has insight.

Reply:
I didn't switch mate packages to meson in fedora, so my knowledge about meson is small.
And i don't think that i ever switch before the whole mate stack use meson.
Mainly people can use meson. If there are errors the will blame us for sure :P
Any way, i found out that mate-control-center is affected too by this issue.